ILSANG/Sources/Views/Ranking/RankingItemView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  RankingItemView.swift
+//  ILSANG
+//
+//  Created by Kim Andrew on 10/7/24.
+//
+
+import SwiftUI
+
+struct RankingItemView: View {
+    
+    var level: Int
+    var nickName: String
+    var stat: String
+    var point: Int
+    
+    var body: some View {
+        HStack (spacing: 17) {
+            RankingProfile(profileImage: nil, level: level)
+            
+            VStack (alignment: .leading, spacing: 3) {
+                Text(nickName)
+                    .font(.system(size: 15))
+                    .fontWeight(.bold)
+                Text("\(stat): \(point)p")
```

Comment:
```suggestion
                Text("\(XpStat(rawValue: stat).headerText): \(point)p")
```

stat을 한글로 표기해야해서요. 
실행해보진 않았는데, 이렇게 수정하면 좋을 것 같습니다ㅏ!